### ISSUE TITLE
Helm installation fails

### ISSUE THREAD

**Issue by dumlutimuralp** [kubernetes-sigs/aws-efs-csi-driver#1330]
/kind bug

**What happened?** When I used helm to install EFS CSI Driver I received the following error : 

```
Warning  Failed     17m (x2 over 17m)     kubelet            Failed to pull image "amazon/aws-efs-csi-driver:v2.0.1": rpc error: code = NotFound desc = failed to pull and unpack image "[docker.io/amazon/aws-efs-csi-driver:v2.0.1](http://docker.io/amazon/aws-efs-csi-driver:v2.0.1)": failed to resolve reference "[docker.io/amazon/aws-efs-csi-driver:v2.0.1](http://docker.io/amazon/aws-efs-csi-driver:v2.0.1)": [docker.io/amazon/aws-efs-csi-driver:v2.0.1](http://docker.io/amazon/aws-efs-csi-driver:v2.0.1): not found
```

It seems that helm values should be updated with the ECR repository path due to [this PR](https://github.com/kubernetes-sigs/aws-efs-csi-driver/pull/1323). It still points out to Docker repo (this part in the Helm [here](https://github.com/kubernetes-sigs/aws-efs-csi-driver/blob/83430aa6abf0bb4be1558d05a65da84605cfaa7b/charts/aws-efs-csi-driver/values.yaml#L12))

**What you expected to happen?** The image to be pulled successfully.

**How to reproduce it (as minimally and precisely as possible)?** Use helm to install EFS CSI Driver

**Anything else we need to know?**:

**Environment** 
- Kubernetes version (use `kubectl version`): Amazon EKS 1.29
- Driver version: 2.0.1

**Please also attach debug logs to help us better diagnose**

- Instructions to gather debug logs can be found [here](https://github.com/kubernetes-sigs/aws-efs-csi-driver/tree/master/troubleshooting)

**Comment by calvinbui** [comment]
i ran into the same issue.

https://github.com/kubernetes-sigs/aws-efs-csi-driver/pull/1323 forgot to change the `image.repository` to `public.ecr.aws/efs-csi-driver/amazon/aws-efs-csi-driver`. 

the tests/qa/ci should've caught this and probably needs to.

**Comment by dongminlee94** [comment]
I am currently experiencing the same issue as reported. I hope it can be resolved quickly.

**Comment by dongminlee94** [comment]
Hi, @mskanth972 

Thank you for the support and the PR addressing the installation issues. I'm pleased to report that the issue has been resolved! By removing and re-adding the Helm repository with the following commands, I was able to successfully deploy the AWS EFS CSI Driver:

```bash
$ helm repo remove aws-efs-csi-driver
$ helm repo add aws-efs-csi-driver https://kubernetes-sigs.github.io/aws-efs-csi-driver
```

This resolved the `ImagePullBackOff` errors I was facing. Thanks again for your prompt assistance and dedication to resolving this issue.
